keystoremanager/keystoremanager.go

```diff
@@ -150,19 +150,20 @@ func (km *KeyStoreManager) GenRootKey(algorithm, passphrase string) (string, err
 	}
 
 	// Changing the root
-	km.rootKeyStore.AddEncryptedKey(privKey.ID(), privKey, passphrase)
+	km.rootKeyStore.AddKey(privKey.ID(), "root", privKey)
 
 	return privKey.ID(), nil
 }
 
-// GetRootCryptoService retreives a root key and a cryptoservice to use with it
-func (km *KeyStoreManager) GetRootCryptoService(rootKeyID, passphrase string) (*cryptoservice.UnlockedCryptoService, error) {
-	privKey, err := km.rootKeyStore.GetDecryptedKey(rootKeyID, passphrase)
+// GetRootCryptoService retrieves a root key and a cryptoservice to use with it
+// TODO(mccauley): remove this as its no longer needed once we have key caching in the keystores
```

Comment:
we're not going to need to the unlocked crypto service once i implement key caching in the keystores, b/c they'll just transparently be retrievable